internal/dynamicserver/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/alecthomas/kong"
+	"google.golang.org/grpc"
+	pref "google.golang.org/protobuf/reflect/protoreflect"
```

Comment:
i wonder whether `preflect` would be better than `pref`. I've seen `pref` before looking though the proto code and it always throws me (to me, pref very strongly means preference). I have not yet remembered what it means and I always go back to the imports to find out. Given that there is a `reflect` package, `preflect` makes more sense to me.

Reply:
same as above. i've read pref a fair bit in the docs now and it entered my brain easily. would prefer not to hold 2 different things. but also same as above: not important enough to me - to have you happy and engaged here nw to change.